xdcs-frontend/src/app/element/log-preview/log-preview.component.html

```diff
@@ -1,6 +1,9 @@
 <pre *ngIf="convert" class="log-output">
   <ng-container *ngFor="let line of lines">
     <div class="log-line">
+      <div class="line-agent">
+        {{ line.agentTag }} + ': '
```

Comment:
po co dwukropek?
jaką szerokość to będzie miało?

Reply:
kwestie samego wyglądu i szerokości dostosuje po merge 172